### PR TITLE
Add openshift/compliance-operator

### PR DESCRIPTION
**Pull request by mrogers950**

### CODE REVIEW

File: ci-operator/config/openshift/compliance-operator/openshift-compliance-operator-master.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+- as: unit
+  commands: make test-unit
+  container:
+    from: src
+- as: e2e-aws
+  commands: make e2e
+  openshift_installer_src:
+    cluster_profile: aws
+promotion:
+  name: "4.3"
```
Comment (petr-muller) [inline ci-operator/config/openshift/compliance-operator/openshift-compliance-operator-master.yaml]: We have cut last week, so these should be 4.4 now

File: ci-operator/config/openshift/compliance-operator/openshift-compliance-operator-master.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  namespace: ocp
+tag_specification:
+  name: "4.3"
```
Comment (petr-muller) [inline ci-operator/config/openshift/compliance-operator/openshift-compliance-operator-master.yaml]: Ditto

File: core-services/prow/02_config/_config.yaml [hunk 3]
```diff
@@ -665,6 +665,7 @@ tide:
     - openshift/cluster-svcat-controller-manager-operator
     - openshift/cluster-update-keys
     - openshift/cluster-version-operator
+    - openshift/compliance-operator
```
Comment (petr-muller) [inline core-services/prow/02_config/_config.yaml]: You should also add the repo to the second query, the one related to release branches (around L462)

**Comment by mrogers950** [comment]
@petr-muller thanks! I've made those changes.

**Comment by petr-muller** [comment]
Sorry for not pointing this out earlier, but I only found out today we have **three** Tide queries now, not two :(

So I think the PR needs to be rebased and the repo should be added to the following three lists:

https://github.com/openshift/release/blob/master/core-services/prow/02_config/_config.yaml#L432
https://github.com/openshift/release/blob/master/core-services/prow/02_config/_config.yaml#L623
https://github.com/openshift/release/blob/master/core-services/prow/02_config/_config.yaml#L813

**Comment by mrogers950** [comment]
@petr-muller I've rebased and added to the sections you highlighted.

**Comment by petr-muller** [comment]
I'm really sorry about this, but I screwed it up in https://github.com/openshift/release/pull/6005#issuecomment-558692696

The repo should be present in exactly three queries in Tide config (now it is in five), these ones (I triple checked). It should not be in any other:

https://github.com/openshift/release/blob/master/core-services/prow/02_config/_config.yaml#L471
https://github.com/openshift/release/blob/master/core-services/prow/02_config/_config.yaml#L661
https://github.com/openshift/release/blob/master/core-services/prow/02_config/_config.yaml#L850

**Comment by mrogers950** [comment]
@petr-muller no problem, I've corrected that and updated.

**Pull request merged**